packages/core/src/org.ts

```diff
@@ -95,9 +95,10 @@ export class OrgMember extends Serializable {
         orgSignature,
         role,
         updated,
+        status,
     }: Partial<OrgMember> = {}) {
         super();
-        Object.assign(this, { accountId, name, email, publicKey, signature, orgSignature, updated });
+        Object.assign(this, { accountId, name, email, publicKey, signature, orgSignature, updated, status });
```

Comment:
I noticed the status wasn't applied correctly (again, an oversight on my part), so I fixed that, too.